js/app/modules/card/list.js

```diff
@@ -54,6 +63,12 @@ const List = new Module.Class({
         if (this.model.duration) {
             this.set_duration_label(this._context_label, this.model.duration);
         }
+
+        if (this.show_context) {
+            this._context_widget = this.create_context_widget_from_model();
+            this._context_frame.add(this._context_widget);
```

Comment:
This actually rely on https://github.com/endlessm/eos-knowledge-lib/blob/master/js/app/moduleFactory.js#L121